### ISSUE TITLE
A11y_code.visualstudio.com_DownloadInsiders_ScreenReader: On interacting with the “Mac”, “Linux” links on the screen the content gets updated but the same is not announced to the screen reader users.

### ISSUE THREAD

**Issue by MayuriNaik** [microsoft/vscode-docs#2826]
“[Check out Accessibility Insights!](https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Faccessibilityinsights.io%2F&data=02%7C01%7Cv-manai%40microsoft.com%7Cb67b2c4b646d4f9561a208d6f4b5c39b%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C636965458847260936&sdata=T26HQfSGOlnuRQdX%2ByXk%2B2bxqgwFvCIVfuboZUWidYY%3D&reserved=0) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

**Environment Details:**
URL: [https://code.visualstudio.com](https://code.visualstudio.com) 
Windows Version: Windows10
Browser Version: 
Microsoft Edge 44.18362.1.0
Chrome Version 75.0.3770.100 (Official Build) (64-bit)
Screen Reader Versions : NVDA2019.1.1
MAS Violated: MAS 4.1.3 (WCAG 4.1.3)

Repro Steps:
Navigate to the url: [https://code.visualstudio.com/insiders/](https://code.visualstudio.com/insiders/) 
follow below steps:
1.	Launch the application Url: [https://code.visualstudio.com](https://code.visualstudio.com) 
2.	Navigate to "Download" button.
3.	Scroll down the page and hit "Insiders build" link.
4.	Navigate to through "Download Insiders" screen.
5.	Start screen reader.
6.	Navigate to the “Mac”, “Linux” links and activate them.
7.	The screen content will get updated and listen if the same has been announced to the screen reader users.

**Actual:**
When screen reader users navigate to the “Mac”, “Linux” links and activate them the content gets updated but the same is not notified to the screen reader users.

**Expected:**
on activating the “Mac”, “Linux” links the content on the screen gets updated the same should be announced to the screen reader users.

**Recommendations:**
Refer below link which is repository of bug fixes code snippets:
[https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx](https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx)

**User Impact:**
If the screen reader users are not notified about the change in the content then the screen reader users might not understand what functionality has taken place on activating the links on the screen.

**MAS Reference:**
MAS 4.1.3 - [https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/Doc.aspx?sourcedoc=%7B684573A7-B089-4131-9B39-0009054125D3%7D&file=MAS%204.1.3%20%E2%80%93%20Status%20Messages.docx&action=default&mobileredirect=true&cid=edf0da59-927e-429e-8fc0-74e9ad175c29 ](https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/Doc.aspx?sourcedoc=%7B684573A7-B089-4131-9B39-0009054125D3%7D&file=MAS%204.1.3%20%E2%80%93%20Status%20Messages.docx&action=default&mobileredirect=true&cid=edf0da59-927e-429e-8fc0-74e9ad175c29 )

**Attachment for Reference:**
[Code.visualstudio.com_DownloadInsiders_ScreenReader Content updated is not announced to screen reader users.pptx](https://github.com/microsoft/vscode-docs/files/3338159/Code.visualstudio.com_DownloadInsiders_ScreenReader.Content.updated.is.not.announced.to.screen.reader.users.pptx)
